src/app/components/LeverageSelector/index.tsx

```diff
@@ -55,11 +60,11 @@ export function LeverageSelector(props: Props) {
         <div className="tw-inline-flex tw-justify-between tw-items-start tw-w-full">
           {items.map(item => (
             <ThemeProvider
-              theme={props.value === item ? active : inactive}
+              theme={value === item ? active : inactive}
               key={item}
             >
               <div>
-                <Button onClick={() => props.onChange(item)} className="btn">
+                <Button onClick={() => onChange(item)} className="btn">
```

Comment:
Don't use lambda, create a handler with `useCallback` and use it here.

Reply:
I committed a version with a callback function, though I don't think it's more readable, and it doesn't fix change detection issues either.

Reply:
It's not about readability, it's more about that I don't like lambda functions in `onClick` because it can cause unnecessary rerenders. And what's the problem with detection? I tried leverage selector and it seems fine.

Reply:
Sorry I meant Reacts change detection that causes rerenders. The new implementation causes equally as many rerenders as a lambda function. I don't know any better solution from the top of my head.